models/__init__.py

```diff
@@ -31,7 +31,7 @@ def get_models_names():
 
         try:
             find_model_using_name(model_name)
-            base_models_to_exclude = ["base", "base_gan", "base_diffusion"]
```

Comment:
If you remove base_gan from here it means that it will appear as selectable model in the options right? not sure that's what we want

Reply:
Oh yes, we don't want that.